src/index.html

```diff
@@ -3,12 +3,37 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="moyoIcon" href="#">
+        <img src="images/logo.png" alt="moyo_icon">
+      </a>
+
+      <nav class="navigationBlock">
+        <ul class="navigationList">
```

Comment:
Use BEM for naming classes
```suggestion
      <nav class="nav">
        <ul class="nav__list">
```